moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -37,67 +30,62 @@ export const PROJECT_LIST_VIEW_QUERY_CONFIG = {
     defaultColumn: "updated_at",
     defaultDirection: "desc",
   },
-  filters: PROJECT_LIST_VIEW_FILTERS_CONFIG,
-  export: PROJECT_LIST_VIEW_EXPORT_CONFIG,
   search: {
     placeholder:
       "Search by ID, name, description, phase, lead, sponsor, partners, eCAPRIS ID...",
     defaultFieldsOperator: "_or",
   },
   columns: {
     project_id: {
-      hidden: true,
-      primary_key: true,
-      searchable: true,
-      sortable: false,
+      searchable: true, // useSearch
+      sortable: true, // useColumns
```

Comment:
Sure! This config is used by the **Signal IDs** column config in the memoized `columns` array for Material Table in `useColumns` (line 280 in `helpers.js`). It is the only column that had `sorting` disabled in the existing code so I updated it to use this k/v pair instead. 

Glad you caught this because it felt weird to have only one column use this config. I could see us adding a `sorting` parameter for all the columns fed to Material Table to be more explicit. Idk - what do you think?

Reply:
ahhh—i'm following now, thanks. i think what you have is a great. the diff was throwing me off but the code is easy to follow 🚢